### PR TITLE
Cleaning up formatting (& delete some reimplemented .js files)

### PR DESCRIPTION
**Pull request by bradleysigma**
`flake8 --max-lin-length=100` returns no violations.
`mypy --ignore-missing-imports --no-strict-optional` returns no violations.
All tests still pass.
Docstrings all use double quotes, on their own lines.
Multi line statements are enclosed in brackets, instead of escaping the newlines.
Multi-line comments are split more evenly.

Jenkins should now run `flake8` and `mypy` on scripts.
image
Also deletes some files from `unimplemented_commands` that have been implemented in Python.

### CODE REVIEW

File: uqcsbot/scripts/trivia.py [hunk 1]
```diff
@@ -62,16 +66,20 @@ def handle_trivia(command: Command):
...
+                             'This command can only be used in private messages with the bot')
             return
 
         first_valid = bot.channels.get(VALID_SEQUETIAL_CHANNELS[0])
-        channel_message = f'Try <#{first_valid.id}|{VALID_SEQUETIAL_CHANNELS[0]}>.' if first_valid else ''
-        bot.post_message(command.channel_id,
-                         f'You cannot use the sequential questions feature in this channel. {channel_message}')
+        channel_message = ''
+        if first_valid:
+            f'Try <#{first_valid.id}|{VALID_SEQUETIAL_CHANNELS[0]}>.'
```
Comment (BraeWebb) [inline uqcsbot/scripts/trivia.py]: This line also no-worky 🙃

File: uqcsbot/scripts/mock.py [hunk 2]
```diff
@@ -29,40 +29,39 @@ def get_nth_most_recent_message(channel_id: str, message_index: int):
...
-    Tries to coerce the message to an integer. If successful, return True-- else
-    returns False.
-    '''
+    """
+    Tries to coerce the message to an integer. If successful, return True else returns False.
+    """
     try:
         int(message)
-    except:
+    except Exception:
```
Comment (BraeWebb) [inline uqcsbot/scripts/mock.py]: Better to use ValueError rather than Exception

File: uqcsbot/scripts/dominos.py [hunk 3]
```diff
@@ -7,40 +7,43 @@
...
+    Returns a list of dominos coupons (default: 5 | max: 10)
+    """
     command_args = command.arg.split() if command.has_arg() else []
 
     parser = argparse.ArgumentParser()
 
     def usage_error(*args, **kwargs):
         raise UsageSyntaxException()
-    parser.error = usage_error # type: ignore
+    parser.error = usage_error  # type: ignore
```
Comment (BraeWebb) [inline uqcsbot/scripts/dominos.py]: Is there a reason for this change?

File: uqcsbot/scripts/dominos.py [hunk 4]
```diff
@@ -7,40 +7,43 @@
...
     def is_valid(self) -> bool:
         try:
             expiry_date = datetime.strptime(self.expiry_date, '%Y-%m-%d')
             now = datetime.now()
-            return expiry_date.year >= now.year and expiry_date.month >= now.month and\
-                expiry_date.day >= now.day
-        except:
+            return (expiry_date.year >= now.year and expiry_date.month >= now.month and
+                    expiry_date.day >= now.day)
+        except Exception:
```
Comment (BraeWebb) [inline uqcsbot/scripts/dominos.py]: I would certainly prefer more specific exceptions

File: uqcsbot/scripts/dominos.py [hunk 5]
```diff
@@ -7,40 +7,43 @@
 from uqcsbot.utils.command_utils import loading_status, UsageSyntaxException
 import requests
 
-MAX_COUPONS = 10 # Prevents abuse
+MAX_COUPONS = 10  # Prevents abuse
```
Comment (BraeWebb) [inline uqcsbot/scripts/dominos.py]: Move this comment above to follow standard

File: uqcsbot/scripts/crisis.py [hunk 6]
```diff
@@ -13,8 +13,8 @@
 @bot.on_command("mentalhealth")
 @bot.on_command("emergency")
 def handle_crisis(command: Command):
-    '''
+    """
     `!crisis`, `!mentalhealth` or `emergency` - Get a list of emergency resources.
```
Comment (BraeWebb) [inline uqcsbot/scripts/crisis.py]: Not your fault but emergency should be prefixed with !

File: uqcsbot/scripts/crates.py [hunk 7]
```diff
@@ -315,16 +345,21 @@ def create_slack_divider_block() -> Dict[str, str]:
...
 
 def get_crate_blocks(crate: CrateResult) -> List[dict]:
-    """Converts a crate into its block based message format for posting to slack"""
+    """
+    Converts a crate into its block based message format for posting to slack
+    """
     return [
-        create_slack_section_block(TextBlock(f'*<{crate.homepage}|{crate.name}>*\n{crate.description}')),
+        create_slack_section_block(TextBlock(f'*<{crate.homepage}|{crate.name}>*'
+                                             f'\n{crate.description}')),
```
Comment (BraeWebb) [inline uqcsbot/scripts/crates.py]: \n should with the string of the line that's being ended

File: uqcsbot/scripts/crates.py [hunk 8]
```diff
@@ -48,41 +50,44 @@ class SlackBlock(ABC):
...
     def __init__(self, text: str, markdown: bool = True):
         self.text = text
         self.markdown = markdown
 
     def get_formatted_block(self):
-        return {
-            'type': 'mrkdwn' if self.markdown else 'plain_text',
-            'text': self.text
-        }
+        return {'type': 'mrkdwn' if self.markdown else 'plain_text', 'text': self.text}
```
Comment (BraeWebb) [inline uqcsbot/scripts/crates.py]: Combining these lines onto the one line obfuscates the meaning. Either split the lines or calculate the value for 'type' prior.

File: uqcsbot/scripts/yt.py [hunk 9]
```diff
@@ -49,17 +49,13 @@ def get_top_video_result(search_query: str, channel):
...
                     developerKey=YOUTUBE_API_KEY, cache_discovery=False)
 
-    search_response = youtube.search().list(
-        q=search_query,
-        part=search_part,
-        maxResults=max_results,
-        type=search_type
-    ).execute()
+    search_response = youtube.search().list(q=search_query, part=search_part,
+                                            maxResults=max_results, type=search_type).execute()
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply uqcsbot/scripts/yt.py]: Made more readable.

File: uqcsbot/scripts/wolfram.py [hunk 10]
```diff
@@ -138,19 +138,16 @@ def wolfram_normal(search_query: str, channel):
     # We also store an identifier string to check against later and the reply_host and s_output
     # string. Attachments is a slack thing that allows the formatting or more complex messages.
     # In this case we add a footer and use the fallback to cheekily store information for later.
-    attachments = [{
-        'fallback': f'WolframCanReply {reply_host} {s_output} {conversation_id}',
-        'footer': 'Further questions may be asked',
-        'text': result,
-    }]
+    attachments = [{'fallback': f'WolframCanReply {reply_host} {s_output} {conversation_id}',
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply uqcsbot/scripts/wolfram.py]: Fixed.

File: uqcsbot/scripts/trivia.py [hunk 11]
```diff
@@ -278,16 +303,24 @@ def add_reaction(reaction: str):
...
-    """Returns the reaction that matches with the correct answer"""
+    """
+    Returns the reaction that matches with the correct answer
+    """
     if question_data.is_boolean:
-        correct_reaction = BOOLEAN_REACTS[0] if question_data.correct_answer == 'True' else BOOLEAN_REACTS[1]
+        if question_data.correct_answer == 'True':
+            correct_reaction = BOOLEAN_REACTS[0]
+        else:
+            BOOLEAN_REACTS[1]
```
[the remark this answers is not in the document]
Comment (bradleysigma) [reply uqcsbot/scripts/trivia.py]: Fixed.

File: uqcsbot/scripts/trivia.py (hunk 1 above)
Comment (bradleysigma) [reply uqcsbot/scripts/trivia.py]: Fixed.

File: uqcsbot/scripts/mock.py (hunk 2 above)
Comment (bradleysigma) [reply uqcsbot/scripts/mock.py]: Fixed.

File: uqcsbot/scripts/dominos.py (hunk 3 above)
Comment (bradleysigma) [reply uqcsbot/scripts/dominos.py]: `flake8` gives [E261](https://lintlyci.github.io/Flake8Rules/rules/E261.html).

File: uqcsbot/scripts/crisis.py (hunk 6 above)
Comment (bradleysigma) [reply uqcsbot/scripts/crisis.py]: Fixed.

File: uqcsbot/scripts/crates.py (hunk 7 above)
Comment (bradleysigma) [reply uqcsbot/scripts/crates.py]: Fixed.

File: uqcsbot/scripts/crates.py (hunk 8 above)
Comment (bradleysigma) [reply uqcsbot/scripts/crates.py]: Made more readable.

File: uqcsbot/scripts/dominos.py (hunk 4 above)
Comment (bradleysigma) [reply uqcsbot/scripts/dominos.py]: Any suggestions?

File: uqcsbot/scripts/dominos.py (hunk 5 above)
Comment (bradleysigma) [reply uqcsbot/scripts/dominos.py]: Comments on the same line are used frequently. Should I change all of them?

File: uqcsbot/scripts/dominos.py (hunk 4 above)
Comment (nicklambourne) [reply uqcsbot/scripts/dominos.py]: Pretty sure you're only expecting ValueErrors from strptime.

File: uqcsbot/scripts/dominos.py (hunk 5 above)
Comment (nicklambourne) [reply uqcsbot/scripts/dominos.py]: Not sure what Brae was referring to. Inline comments are fine as long as there are two spaces before the hash.

File: uqcsbot/scripts/trivia.py [hunk 12]
```diff
@@ -156,7 +171,10 @@ def handle_question(channel: Channel, args: argparse.Namespace):
 
     # Get the answer message
     if question_data.is_boolean:
-        answer_text = f':{BOOLEAN_REACTS[0]}:' if question_data.correct_answer == 'True' else f':{BOOLEAN_REACTS[1]}:'
```
Comment (nicklambourne) [inline uqcsbot/scripts/trivia.py]: Why do you hate ternary operators so much?

Comment (bradleysigma) [reply uqcsbot/scripts/trivia.py]: `uqcsbot/scripts/trivia.py:159:101: E501 line too long (118 > 100 characters)`

File: uqcsbot/scripts/dominos.py (hunk 4 above)
Comment (bradleysigma) [reply uqcsbot/scripts/dominos.py]: Fixed.

File: uqcsbot/scripts/trivia.py (hunk 12 above)
Comment (bradleysigma) [reply uqcsbot/scripts/trivia.py]: Also, SQL's `between` is a ternary operator; FORTH's `*/` is a ternary operator.

**Comment by nicklambourne** [comment]
Hey @bradleysigma can you merge current master into this to resolve the conflicts with !parking?

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
Sallinger

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
retest this please

**Comment by bradleysigma** [comment]
https://api.uqfinal.com

**Comment by bradleysigma** [comment]
retest this please

**Comment by nicklambourne** [comment]
Are you happy with how this is now @bradleysigma?

**Comment by bradleysigma** [comment]
It seems as though `!events` is now working consistently, so yes.

**Comment by nicklambourne** [comment]
Goddamn it, I knew I should have merged this one first. Gonna need you to bring it up to parity and I'll just merge it asap.

**Pull request merged**